sys/net/link_layer/ieee802154/include/ieee802154_config.h

```diff
@@ -0,0 +1,54 @@
+/**
+* @defgroup config_ieee802154 IEEE802.15.4 configurations
+* @brief   Default IEEE802.15.4 values for devices to choose
+* @ingroup net_ieee802154
+* @ingroup configurations
+* @{
+*/
+
+/**
+* @name    IEEE802.15.4 sub-GHZ channel.
+* @{
```

Comment:
Having a group for just one macro seems overkill. Why not just use `@brief` and no group (or the `@brief` shorthand `/**< ... */`?

Reply:
yes, sure. We can do that